mathcomp/_CoqProject

```diff
@@ -5,3 +5,5 @@
 -arg -w -arg -redundant-canonical-projection
 -arg -w -arg -notation-overridden
 -arg -w -arg -duplicate-clear
+-arg -w -arg -ambiguous-paths
```

Comment:
IMO, we should be aware of ambiguous paths with regard to packed classes. Also, I'm trying to reduce the number of ambiguous paths reported so that we can remove this from `mathcomp/Make` (see #297, coq/coq#11258, and coq/coq#11381).